client/components/logged-out-form/links.jsx

```diff
@@ -1,24 +1,13 @@
 import clsx from 'clsx';
-import { omit } from 'lodash';
 import PropTypes from 'prop-types';
-import { Component } from 'react';
 
 import './links.scss';
 
-export default class LoggedOutFormLinks extends Component {
-	static propTypes = {
-		children: PropTypes.node.isRequired,
-		className: PropTypes.string,
-	};
-
-	render() {
-		return (
-			<div
-				{ ...omit( this.props, 'classNames' ) }
```

Comment:
Was this a typo? `classNames` vs `className`. It didn't matter much because we were overriding `className` on the very next line anyway.